features/azure/file.include/etc/systemd/system/chronyd-restricted.service.d/05-private-devices-access.conf

```diff
@@ -0,0 +1,4 @@
+[Service]
+DeviceAllow=/dev/ptp_hyperv
+DevicePolicy=closed
```

Comment:
setting this to `closed` might not even be necessary, as according to the docs[^1] `auto` (the default value) will behave the same, if any `DeviceAllow` value is present.

But this makes it a bit clearer, so probably good.

[^1]: https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html#auto